### ISSUE TITLE
Support wildcard table exclusions

### ISSUE THREAD

**Issue by bomoko** [amazeeio/lagoon-sync#28]
Drush's sql-sync has the ability to exclude tables based on patterns.

A new feature that could, for example, all tables with the structure `cache_*` is required for lagoon-sync to have parity here.

**Comment by seanhamlin** [comment]
This feature has been requested by a customer, basically the same functionality as the drush command, so structure only on a table pattern.

**Comment by timclifford** [comment]
@seanhamlin so we are talking about replicating the `structure-tables-list` that drush supports? essentially returns the structure, but no data?

Do we want to expand on this issue and allow include a `--no-data` option as well?https://dev.mysql.com/doc/refman/8.0/en/mysqldump.html#option_mysqldump_no-data

**Comment by seanhamlin** [comment]
Correct, an example use case is to do structure only for tables that match

* `cache`
* `cache_%`
